ts/src/profiler.ts

```diff
@@ -267,6 +267,7 @@ export class Profiler extends common.ServiceObject {
       uri: '/profiles',
       body: reqBody,
       json: true,
+      timeout: 60 * 60 * 1000,
```

Comment:
You don't have new tests in the PR, so presumably some manual testing was done to verify the fix. Can you describe the testing you did in the PR description?

Reply:
done.